apps/api/src/app/events/usecases/parse-event-request/parse-event-request.command.ts

```diff
@@ -14,8 +14,8 @@ export class ParseEventRequestCommand extends EnvironmentWithUserCommand {
   @IsDefined()
   overrides: Record<string, Record<string, unknown>>;
 
-  @IsDefined()
-  to: TriggerRecipients;
+  @IsOptional()
+  to?: TriggerRecipients;
```

Comment:
Was debating with myself is to update the `to`, to be able to get AddressingTypeEnum as well or not.
in the end decided to add addressingType.
Will love to hear if anyone has any ideas regarding the new Command interface.